packages/ui/src/tooltip/Tooltip.css

```diff
@@ -0,0 +1,24 @@
+.ui-tooltip-aria {
```

Comment:
I've been capitalizing where the component/block/element name appears in the class (to match the capitalization of React components), and then using a BEM style naming method. You can see more info about it in the README file at the project root.

examples for here:
.ui-Tooltip--Aria
.ui-Tooltip--Content
.ui-Tooltip--Triangle